src/ucp/wireup/select.c

```diff
@@ -948,18 +957,21 @@ static double ucp_wireup_aux_score_func(ucp_context_h context,
 {
     /* best end-to-end latency and larger bcopy size */
     return (1e-3 / (ucp_wireup_tl_iface_latency(context, iface_attr, remote_iface_attr) +
-            iface_attr->overhead + remote_iface_attr->overhead)) +
-           (1e3 * ucs_max(iface_attr->cap.am.max_bcopy, iface_attr->cap.am.max_short));
```

Comment:
after this change, wireup transport will be selected only according to latency, and not the maximal size of am_bcopy. which i think should have been done in the first place.